client/OPUNetGameSelectWnd.h

```diff
@@ -55,7 +55,7 @@ class OPUNetGameSelectWnd : public IDlgWnd
 	UINT_PTR timer = 0;
 	UINT searchTickCount = SearchTickInterval - 1;	// Broadcast right away
 	HostedGameInfo* joiningGame = 0;
-	char joinRequestPassword[16];
+	char joinRequestPassword[16] = { 0 };
```

Comment:
I think `{}` may be sufficient to zero initialize.

Reply:
I switched it over. Based on my reading I think they are identical in effect for an array initialization. `{ 0 }` is C compliant if that matters in a given code base.